spec/system/page_objects/pages/category.rb

```diff
@@ -20,6 +20,23 @@ def visit_edit_template(category)
         self
       end
 
+      def visit_categories
+        page.visit("/categories")
+        self
+      end
+
+      def visit_new_category
+        page.visit("/new-category")
+        self
+      end
+
+      def focus_input(selector)
+        execute_script(<<~JS, text)
+          document.querySelector("#{selector}").focus();
```

Comment:
I think to avoid having to do this you can focus on an element by clicking on it:

https://github.com/discourse/discourse/blob/0b3b101c97b6a027ee11bcd166d1ab49a63a0c00/plugins/chat/spec/system/page_objects/chat/components/composer.rb#L97-L99